CHANGELOG.md

```diff
@@ -172,6 +172,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix for stuck update action in a bulk with `retry_on_conflict` property ([#11152](https://github.com/opensearch-project/OpenSearch/issues/11152))
 - Remove shadowJar from `lang-painless` module publication ([#11369](https://github.com/opensearch-project/OpenSearch/issues/11369))
 - Fix remote shards balancer and remove unused variables ([#11167](https://github.com/opensearch-project/OpenSearch/pull/11167))
+- Replication type configuration preference for indices creating using templates ([#11417](https://github.com/opensearch-project/OpenSearch/pull/11417))
```

Comment:
Nitpick, but make this an imperative statement, like "Support setting replication type in index templates" or the PR title "Fix template setting override for replication type"